Fuga.php

```diff
@@ -8,10 +8,11 @@
 class Fuga {
 
     public function index() {
-        return true;
+        $i = 0;
+        if($i == 1)
```

Comment:
__[code_sniffer]__
* Expected 1 space after IF keyword; 0 found
* Inline control structures are not allowed